components/dropdown-menu/index.js

```diff
@@ -60,47 +79,58 @@ class DropdownMenu extends wp.element.Component {
 	}
 
 	findActiveIndex() {
-		if ( this.menuRef ) {
-			const menu = findDOMNode( this.menuRef );
+		if ( this.menuNode ) {
 			const menuItem = document.activeElement;
-			if ( menuItem.parentNode === menu ) {
-				return findIndex( menu.children, ( child ) => child === menuItem );
+			if ( menuItem.parentNode === this.menuNode ) {
+				return findIndex( this.menuNode.children, ( child ) => child === menuItem );
 			}
 			return -1;
 		}
 	}
 
 	focusIndex( index ) {
-		if ( this.menuRef ) {
-			const menu = findDOMNode( this.menuRef );
+		if ( this.menuNode ) {
 			if ( index < 0 ) {
```

Comment:
With changes to `focusPrevious`, is it ever the case that this condition would pass? Also generally concerned about `previousElementSibling` potentially not being a focusable element. Might be worth dropping this.